shared/src/address/default.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019-2022 ChainSafe Systems
```

Comment:
Let's just put this in the `network.rs` module. An `address::default` module doesn't really make sense.

Reply:
fixed